docs/project/governance.md

```diff
@@ -0,0 +1,97 @@
+# pyodide governance and decision-making
+
+The purpose of this document is to formalize the governance process used by the
+pyodide project, to clarify how decisions are made and how the various
+elements of our community interact.
+This document establishes a decision-making structure that takes into account
+feedback from all members of the community and strives to find consensus, while
+avoiding any deadlocks.
+
+Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the {ref}`how_to_contibute`.
```

Comment:
Would be nice to explicitly mention e.g., answering user questions as a way to contribute here.

Reply:
Absolutely, but I think different ways of how to contribute, such as answering user questions, reviewing PRs should be listed in https://pyodide.readthedocs.io/en/latest/development/contributing.html not in the governance. That page needs re-organizing anyway.

For instance an example of such section in https://scikit-learn.org/stable/developers/contributing.html#ways-to-contribute

Reply:
I think giving a single example like "answering user questions" here might help indicate that we mean "contribute" in a very inclusive sense.